tests/wp-unit/include/Core/Users/Modules/DeleteUsersByUserMetaModuleTest.php

```diff
@@ -158,7 +158,7 @@ public function test_delete_users_by_meta_with_filter_set_post_type() {
 			'registered_restrict' => false,
 			'login_restrict'      => false,
 			'no_posts'            => true,
-			'no_posts_post_types' => 'post',
+			'no_posts_post_types' => array( 'post' ),
```

Comment:
The code changes are fine, but the commit message format is wrong.

Refer to https://github.com/sudar/bulk-wp-site/wiki/Workflow#git-commit-messages for instructions on the commit message format.